src/isar_exr/api/energy_robotics_api.py

```diff
@@ -66,3 +66,46 @@ def pause_current_mission(self, exr_robot_id: str) -> None:
         ]
         if not success:
             raise RobotException(f"Invalid status after pausing mission: '{status}'")
+
+    def create_mission_definition(
+        self, site_id: str, mission_name: str, exr_robot_id: str
+    ) -> str:
+        mutation_string: str = """
+            mutation createMissionDefinition($site_id:String!, $mission_name:String!, $exr_robot_id:CreateRequiredRobotConfigInput) {
+                createMissionDefinition(
+                input: { siteId: $site_id, name: $mission_name, requiredRobotConfig: $exr_robot_id }
+                ) {
+                id
+            }}
+        """
+        params: dict = {
+            "site_id": site_id,
+            "mission_name": mission_name,
+            "exr_robot_id": exr_robot_id,
+        }
+
+        try:
+            response_dict: dict[str, Any] = self.client.query(mutation_string, params)
```

Comment:
is it sufficient to say dict?

Reply:
dont know what you mean. It's the same as the other functions we have made. Il just let it stand and we can update it later maby